documentation/modules/ROOT/pages/_attributes.adoc

```diff
@@ -11,3 +11,9 @@
 :install-version: 0.10
 :confluent-platform-version: 5.1.2
 :strimzi-version: 0.9.0
+
+:prodname: Debezium
+:assemblies: ../assemblies
+:modules: ../../modules
+
+:mysql-connector-plugin-download: https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/1.0.0.Beta2/debezium-connector-mysql-1.0.0.Beta2-plugin.tar.gz
```

Comment:
The version should refer to the "debezium-version" variable instead of hard-coding it.